package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "module",
```

Comment:
made all js into a module to use `import` and `export`. Not sure if this is the correct way or not